### PR TITLE
Bestman xrootd SE

### PR DESCRIPTION
**Pull request by efajardo**
https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallBestmanXrootdSE#Screen_Dump_of_Install_Procedure

- [x] Enter date into "Migrated" column of google sheet
- [x] Add migration header to TWiki document

### CODE REVIEW

**Comment by efajardo** [comment]
@matyasselmeci can you please review?

File: docs/data/install-bestman-xrootd.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
+Install Bestman Xrootd SE
+=========================
+
+About this Document
+===================
+
+This page explains how to install the BeStMan Storage Element with underlying XRootD storage.
+
+!!! note
```
Comment (bbockelm) [inline docs/data/install-bestman-xrootd.md]: Remove this note.

File: docs/data/install-bestman-xrootd.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+You will also need a copy of CA certificates (see below).
+
+Networking
+----------
+
+| Service Name            | Protocol    | Port Number                      | Inbound | Outbound |Commen                     |
+|:------------------------|:------------|:---------------------------------|:--------|:---------|:--------------------------|
+| GridFTP                 | tcp         |2811 and `GLOBUS_TCP_SOURCE_RANGE`| YES     |          | contiguous range of ports |
+| Storage Resource Manager| tcp         | 8080                             | YES     |          |                           |
```
Comment (bbockelm) [inline docs/data/install-bestman-xrootd.md]: Nope, we shouldn't be listening on this.

File: docs/data/install-bestman-xrootd.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+---------------------
+
+See [Bestman Instructions](../data/bestman-install/#modify-etcsudoers)
+
+### (Optional) Copying certificates to a bestman location
+
+See [Bestman Instructions](../data/bestman-install/#copying-certificates-to-an-alternate-location)
+
+
+Configure Xrootd Gratia Probes
```
Comment (bbockelm) [inline docs/data/install-bestman-xrootd.md]: I don't really think any of this works anymore.  I would remove the whole section unless we see some motivation to add it back.

File: docs/data/install-bestman-xrootd.md (hunk 1 above)
Comment (efajardo) [reply docs/data/install-bestman-xrootd.md]: Done

File: docs/data/install-bestman-xrootd.md (hunk 2 above)
Comment (efajardo) [reply docs/data/install-bestman-xrootd.md]: Fixed

**Pull request merged**